components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/APIEntity.java

```diff
@@ -79,6 +79,14 @@ public class APIEntity {
     @TypeConverter(name = "uuidConverter", dataType = Object.class, objectType = UUID.class)
     private UUID formationID;
 
+    @ElementCollection
+    @CollectionTable(name = "ord_hierarchy_api_definitions", joinColumns = @JoinColumn(name = "api_definition_id"))
+    private List<ArrayElement> hierarchy;
```

Comment:
With this PR you introduce the newly added fields to the ORD Service by defining them in the given class:

ProductEntity-> tags
VendorEntity -> tags
SystemInstanceEntity -> tags
BundleEntity -> tags
APIEntity -> hierarchy, supported_use_cases
EventEntity -> hierarchy, supported_use_cases

So far so good.

These newly added fields are `@Embeddable` and are defined in the 'parent' class with `@ElementCollection
  @CollectionTable(...)`
  
  In order to be able to query these new `@Embeddable` fields via the OData API (e.g. query the `hierarchy` of the API): `/apis?$filter=hierarchy/any(<someCondition>))` you will have to create a 'helper entity'. More info about the querying aspect and why do we need a 'helper entity' can be found [here](https://github.com/SAP/olingo-jpa-processor-v4/issues/109).
  
So, what helper entities have to be added in this case. First of all, [here](https://github.com/kyma-incubator/ord-service/tree/0d2f533c9a5d5633341ccce7c7e9ca07adfba4f0/components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/helperentities) are all current 'helper entities' which pattern we will follow. You will have to create a separate class in this directory for:
- `TagProduct.java` (corresponding to the added `tags` field in the `Product` class)
- `TagVendor.java` (corresponding to the added `tags` field in the `Vendor` class)
- `TagBundle.java` (corresponding to the added `tags` field in the `Bundle` class)
- `TagSystemInstance.java` (corresponding to the added `tags` field in the `SystemInstance` class)
- `HierarchyAPIDefinition.java` (corresponding to the added `hierarchy` field in the `API` class)
- `SupportedUseCasesAPIDefinition.java` (corresponding to the added `supported_use_cases` field in the `API` class)
- `HierarchyEventDefinition.java` (corresponding to the added `hierarchy` field in the `Event` class)
- `SupportedUseCasesEventDefinition.java` (corresponding to the added `supported_use_cases` field in the `Event` class)

and adjust the table names to correspond to the correct view you created in the compass migration. You can use the existing classes for reference. The new ones will have the same structure.